sources/views/contactsData/contactsList.js

```diff
@@ -0,0 +1,94 @@
+import {JetView} from "webix-jet";
+import {contacts} from "../../models/contacts";
+import {statuses} from "../../models/statuses";
+
+export default class ContactsListView extends JetView {
+	config() {
+		return {
+			rows: [
+				{
+					view: "list",
+					localId: "contactsList",
+					width: 300,
+					type: {
+						height: "auto"
+					},
+					template: obj => `
+									<div class="contact">
+										<div class="contact_avatar">
+											<i class="mdi mdi-36px mdi-account-circle"></i>
+										</div>
+										<div class="contact_details">
+											<div class="contact_details_div">
+												<b>${obj.FirstName || " - "} ${obj.LastName || " - "}</b>
+											</div>
+											<small class="contact_details_div">${obj.Company || " - "}</small>
+										</div>
+									</div>`,
+					select: true,
+					scroll: "y",
+					on: {
+						onAfterSelect: (id) => {
+							this.setParamToUrl(id);
+							let contactInfo = contacts.getItem(id);
+							const contactStatusId = contactInfo.StatusID;
+							const item = statuses.getItem(contactStatusId);
+							if (contactStatusId && item) {
+								contactInfo.Status = statuses.getItem(contactStatusId).Value;
+							}
+							this.app.callEvent("onChangeUsersListUrl", [contactInfo]);
+						}
+					}
+				}
+			]
+		};
+	}
+
+	init(url) {
+		this.list = this.$$("contactsList");
+		webix.promise.all([
+			contacts.waitData,
+			statuses.waitData
+		]).then(() => {
+			this.list.sync(contacts);
+			let id = url[0].params.id;
```

Comment:
It's not recommended unless you have no other options and sure that the structure of your app is constant enough to expect the correct result.
Use [getParam](https://webix.gitbook.io/webix-jet/api/jetview-api/jetview-methods#this-getparam) instead of direct reference to the URL where it's possible.

Reply:
fixed

Reply:
fixed